tests/xfstests/install.pm

```diff
@@ -31,9 +31,14 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
+    # Disable PackageKit
+    pkcon_quit;
```

Comment:
Why is this needed? Could you just add a few comment on this?

Reply:
This is needed because, sometimes, PackageKit is running and zypper cannot run. 

The `pkcon_quit` is performed by `prepare_for_kdump` (called by `xfstests/enable_kdump.pm`), but this module is not run if the **NO_KDUMP** variable is set.